backend/internal/storage/job_offers_db.go

```diff
@@ -6,6 +6,59 @@ import (
 	"gorm.io/gorm"
 )
 
+func (db DB) GetJobOffers(page string, limit string, jobtitle string, company string, city string, isRemote string) (v1beta.JobOffersResponse, error) {
+	offset, limitInt := Paginate(page, limit)
+	var nbHits int64
+
+	// Build the query
+	query := db.queryJobOffers().Order("jb.createdat DESC")
+
+	if jobtitle != "" {
+		query = query.Where("j.title LIKE ?", "%"+jobtitle+"%")
+	}
+
+	if company != "" {
+		query = query.Where("c.name LIKE ?", "%"+company+"%")
+	}
+
+	if city != "" {
+		query = query.Where("ct.name LIKE ?", "%"+city+"%")
+	}
+
+	switch isRemote {
+	case "true":
+		query = query.Where("jb.seniority = ?", true)
+		break
+	case "false":
+		query = query.Where("jb.seniority = ?", false)
+		break
+	}
```

Comment:
```suggestion
        query = query.Where("jb.seniority = ?", isRemote)
```

? 🤔

Reply:
If you look carefully, `isRemote` is not a boolean but a string, so I guess to do what you suggest here he may have to change that. I also think we should use a boolean instead of a string here. I will see if I can change that

Reply:
After reading at the code it seems like we have no choice but to do it this way, as the `q.IsRemote` field comes from the url request query. So the cast has to be done somewhere and it is fine for it to be done in this file. I will go ahead and merge this PR.

Reply:
alright 👍🏽